<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WidgetForListWidgetBegin</class>
 <widget class="QWidget" name="WidgetForListWidgetBegin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>180</width>
    <height>43</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#WidgetForListWidgetBegin
{
	background-color: rgb(255, 255, 255);
}
QWidget#WidgetForListWidgetBegin:hover
{
	background-color: rgb(214, 255, 117);
}
QWidget#WidgetForListWidgetBegin:pressed
{
	background-color: rgb(255, 255, 255);
}</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>5</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="horizontalSpacing">
    <number>5</number>
   </property>
   <property name="verticalSpacing">
    <number>0</number>
   </property>
   <item row="0" column="0" rowspan="2">
    <widget class="QLabel" name="labelImage">
     <property name="minimumSize">
      <size>
       <width>44</width>
       <height>39</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>44</width>
       <height>39</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLabel" name="labelText">
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>-1</pointsize>
       <weight>11</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 90 14px &quot;Arial&quot;;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLabel" name="labelType">
     <property name="styleSheet">
      <string notr="true">color: rgb(127, 127, 127);
font: 12px &quot;arial&quot;;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
